debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/SnapshotReader.java

```diff
@@ -131,14 +133,20 @@ protected void doCleanup() {
      */
     protected void execute() {
         context.configureLoggingContext("snapshot");
-        logger.info("Starting snapshot");
         final AtomicReference<String> sql = new AtomicReference<>();
         final JdbcConnection mysql = context.jdbc();
         final MySqlSchema schema = context.dbSchema();
         final Filters filters = schema.filters();
         final SourceInfo source = context.source();
         final Clock clock = context.clock();
         final long ts = clock.currentTimeInMillis();
+        logger.info("Starting snapshot for {} with user '{}' and pass '{}' at {}", context.connectionString(), mysql.username(),
+                    mysql.password(), new Date(clock.currentTimeInMillis()));
+        if (logger.isDebugEnabled()) {
+            logServerInformation(sql, mysql);
+            logRolesForCurrentUser(sql, mysql);
+        }
+        
```

Comment:
This is awesome. I'm thinking the server information should be logged at INFO level, and wondering if the roles should be as well.